Sources/WalletConnectUtils/JsonRecID.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public struct JsonRpcID {
+
+    public static func generate() -> Int64 {
+        let timestamp = Int64(Date().timeIntervalSince1970 * 1000) * 1000
```

Comment:
we probably should change this for real milliseconds as this are seconds multiplied by 1000 now.

Reply:
it is likely to happen that there will be two requests send in one seconds and there is 50% chance that random suffix will be greater in first request. if it happens, second request will not update the session even though it should.